guides/index-template.md

```diff
@@ -0,0 +1,304 @@
+# Index Template
+Index templates allow you to define default settings, mappings, and aliases for one or more indices during their creation. This guide will teach you how to create index templates and apply them to indices using the OpenSearch .NET client.
+
+## Setup
+**This guide is designed specifically for the OpenSearch.Net low-level client. At the time this guide was written, none of the methods referenced in the examples below exist yet in the high-level OpenSearch.Client. These examples can be used as a model for the time at which the high-level client implementations are being developed.**
```

Comment:
This can be simplified:
```suggestion
**At the time of writing the API methods related to composable templates do not yet exist in the high-level client, as such this guide makes use of their low-level counterparts.**
```